### DISCUSSION TITLE
Request for refund due to purchase of Chrome license that does not allow video downloading

### DISCUSSION THREAD

**Discussion by Minhyuk1233** [aclap-dev/video-downloadhelper#1485] [License & Purchase requests]
### What's your issue?

You said you could download YouTube videos, so I paid, but I couldn't download them in the Chrome environment? I thought of course the license key would be connected. 
The license doesn't work in Firefox again.

I strongly want a refund and have sent this to your email as well.

Please request a quick refund, and if the refund has been received, please reply to my email for the refund.

P.S.
If possible, deactivate my Chrome license key and activate the Firefox license.

### The 10 first digits (only 10) of your license number or email address

e9124db29e

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
